feature: more option for draw-controls text

johnnyblasta opened the pull request:
Closes #1950

Grammostola commented:
A nice addition. I noticed two things:
- Rotate resets though its slider does not whenever another prop is changed 
- If draw is activated and a point is drawn first thing the map starts panning with the cursor

johnnyblasta commented:
> A nice addition. I noticed two things:
> 
> * Rotate resets though its slider does not whenever another prop is changed
> * If draw is activated and a point is drawn first thing the map starts panning with the cursor. This does not appear to happen in the current master.

Fixed the problem with point drawn first.
Removed the rotate setting to zero since saw no good way of accomplish what I had actually intended to happen. At least doing it from the style window, could have done something with the drawtools and drawhandler perhaps.
